fwm-docs-qs/azuredeploy.json

```diff
@@ -0,0 +1,464 @@
+           "apiVersion": "2019-11-01",
+           "name": "publicIP-jump-srv",
+           "location": "[resourceGroup().location]",
+           "sku": {
+               "name": "Standard"
+           },
+           "properties": {
+               "publicIPAddressVersion": "IPv4",
+               "publicIPAllocationMethod": "Static",
+               "idleTimeoutInMinutes": 4
+           }
+       },
+        {
+            "type": "Microsoft.Network/routeTables",
+            "apiVersion": "2020-03-01",
+            "name": "RT-01",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "disableBgpRoutePropagation": false,
+                "routes": [
+                    {
+                        "name": "jump-to-inet",
+                        "properties": {
+                            "addressPrefix": "0.0.0.0/0",
+                            "nextHopType": "Internet"
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "type": "Microsoft.Network/virtualHubs/routeTables",
+            "apiVersion": "2019-07-01",
+            "name": "Hub-01/VirtualNetworkRouteTable",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                "routes": [
+                {
+                   "destinationType": "CIDR",
+                        "destinations": [
+                            "10.0.1.0/24",
+                            "0.0.0.0/0"
+                                        ],
+                         "nextHopType": "IPAddress",
+                            "nextHops": [
+                                "10.1.64.4"
+                                        ]
+                                    }
+                 ],
+                "attachedConnections": [
+                    "All_Vnets"
+                ]
+            },
+            "dependsOn": [
+                "[resourceId('Microsoft.Network/virtualHubs', 'Hub-01')]",
+                "[resourceId('Microsoft.Network/azureFirewalls', 'AzfwTest')]"
+            ]
+        },
+        {
+            "type": "Microsoft.Network/virtualHubs/routeTables",
+            "apiVersion": "2019-07-01",
+            "name": "Hub-01/BranchRouteTable",
+            "location": "[resourceGroup().location]",
+            "properties": {
+                    "routes": [
+                    {
+                        "destinationType": "CIDR",
+                           "destinations": [
+                               "10.0.1.0/24"
+                            ],
+                        "nextHopType": "IPAddress",
+                            "nextHops": [
+                                "10.1.64.4"
+                            ]
+                    }
+                    ],
+                        "attachedConnections": [
+                           "All_Branches"
+                        ]
+                },
+                "dependsOn": [
```

Comment:
move all dependsOn properties above the properties property

Reply:
done